docs/scripts/generate-sitemap-robotstxt.mjs

```diff
@@ -24,33 +24,68 @@ async function generateSitemap() {
     '!src/pages/404.page.tsx',
   ]);
 
-  const pages = pagesWithParam.flatMap((p) =>
-    ['react', 'angular', 'vue', 'flutter'].map((framework) =>
-      p.replace('[platform]', framework)
-    )
-  );
+  const pages = pagesWithParam.flatMap((p) => {
+    p = p
+      .replace('src/pages', '')
+      .replace('data', '')
+      .replace('.js', '')
+      .replace('.page.mdx', '')
+      .replace('.page.tsx', '')
+      .replace('/index', '');
+    return ['react', 'angular', 'vue', 'flutter'].map((framework) => {
+      const frameworkWithAllPath = 'react';
+      const navNotForAllFrameworks = '[platform]/components';
+      const pathForAllFrameworks = [
+        '/[platform]/components',
```

Comment:
Won't this exclude /getting-started and /theming paths?

Reply:
Refactored to use `supportedFramework` in frontmatter